Philips.CodeAnalysis.Test/MsTest/DataTestMethodsHaveDataRowsAnalyzerTest.cs

```diff
@@ -79,7 +79,7 @@ public class Tests
 		[TestCategory(TestDefinitions.UnitTests)]
 		public void DataTestMethodsMustHaveDataRows2(string arg, bool isError)
```

Comment:
Let's split this method also,these isError parameters are confusing

Reply:
Round 2